docs/product-installation-guide/src/main/asciidoc/jws-zip-install-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='jws-zip-install-proc']
+
+= Installing {KIE_SERVER} on Red Hat JBoss Web Server ZIP installation
+
+
+{KIE_SERVER} provides the runtime environment for business assets and accesses the data stored in the assets repository (knowledge store). This section explains how to perform the ZIP file installation to install {KIE_SERVER} on an existing Red Hat JBoss Web Server 3.1 instance.
+
+[NOTE]
+====
+To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing-using-installer>>.
+====
+
+.Prerequisites
+* A backed up Red Hat JBoss Web Server 3.1 with Tomcat 8 or higher server installation. The base directory of the JBoss Web Server installation is referred to as `__JWS_HOME__`. 
+* Sufficient user permissions to complete the installation.
+* The `rhdm-7.0-kie-server-jws.zip` file. To download this file:
+
```

Comment:
These are not being rendered as numeric steps (1 and 2) because I believe you need to add an asciidoc inclusion for it, like this:
````
...To download this file:
+
--
. Click ...
. Unzip ...
--
+
.Procedure
````
Or something like that. Maybe even just +. Not sure, but worth trying a few things to get it right. Not the end of the world if not, I suppose.